CSharpSourceCode/Utilities/Extensions/CampaignExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TaleWorlds.CampaignSystem;
+
+namespace TOW_Core.Utilities.Extensions
+{
+    public static class CampaignExtensions
+    {
+        public static List<CharacterObject> GetTOWTemplates(this Campaign campaign)
+        {
+            List<CharacterObject> list = new List<CharacterObject>();
+            foreach(var item in campaign.TemplateCharacters)
```

Comment:
MBReadOnlyList implements IEnumerable, so it should be possible to use LINQ instead of foreach

Reply:
fixed